mibidata/tiff.py

```diff
@@ -211,8 +217,12 @@ def write(filename, image, sed=None, optical=None, ranges=None,
                 'channel.mass': int(image.masses[i]),
                 'channel.target': image.targets[i],
             })
-            page_name = (285, 's', 0, '{} ({})'.format(
-                image.targets[i].encode(), image.masses[i]))
+            # Converting to bytes string to support non-ascii characters
```

Comment:
Since we repeat this code twice, should it be in a function?

Reply:
Done